recipes/r-stochvol/meta.yaml

```
@@ -0,0 +1,95 @@
+{% set version = '3.2.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-stochvol
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/stochvol_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/stochvol/stochvol_{{ version }}.tar.gz
+  sha256: 5c149912cbb774c0baccc62ee6b78fcdae378595d143a5a17c7bb56f89cd28b3
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-rcpp >=1.0
+    - r-rcpparmadillo >=0.9.900
+    - r-coda >=0.19
+    - libblas
+    - liblapack
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-rcpp >=1.0
+    - r-rcpparmadillo >=0.9.900
+    - r-coda >=0.19
+
+test:
+  commands:
+    - $R -e "library('stochvol')"           # [not win]
+    - "\"%R%\" -e \"library('stochvol')\""  # [win]
+
+about:
+  home: https://gregorkastner.github.io/stochvol/
+  dev_url: https://github.com/gregorkastner/stochvol
+  license: GPL-2.0-only
```

License shows `GPL (>= 2)`, so:

```suggestion
  license: GPL-2.0-or-later
```